### PR TITLE
[Nu-1892] add missing tooltips

### PR DESCRIPTION
**Pull request by Dzuming**
## Describe your changes
- Scenario details category
image

- Scenario details name
image

- Scenario details label
image

- Scenarios list avatar
image
image

- Scenarios list category
image

- Scenarios list name (it's already there)
image

- Scenarios list status
image

- Scenarios list processing mode
image

- Scenarios list label
image

## Checklist before merge
- [x] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [ ] Code is cleaned from temporary changes and commented out lines
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

## Release Notes

- **New Features**
	- Enhanced internationalization support across various components, allowing for localized titles and tooltips based on user language settings.
	- Added a new Activities panel consolidating scenario activities into a single chronological list.
	- Introduced scenario labels for improved organization and retrieval of scenarios.

- **Bug Fixes**
	- Various user experience improvements, including a revamped UI for aggregation definitions and enhanced validation handling.

- **Documentation**
	- Updated changelog to reflect new features, improvements, and fixes.

### CODE REVIEW

File: designer/client/src/components/toolbars/scenarioDetails/ScenarioLabels.tsx [hunk 1]
```diff
@@ -352,6 +354,7 @@ export const ScenarioLabels = ({ readOnly }: Props) => {
                                 const labelError = labelOptionsErrors.find((error) => error.label === toLabelValue(option));
                                 return (
                                     <StyledLabelChip
+                                        title={t("panels.scenarioDetails.labels.title", "Label")}
```
[the remark this answers is not in the document]
Comment (arkadius) [reply designer/client/src/components/toolbars/scenarioDetails/ScenarioLabels.tsx]: @Dzuming I agree on that. Only "Label" might by confusing.

[the remark this answers is not in the document]
Comment (arkadius) [reply designer/client/src/components/toolbars/scenarioDetails/ScenarioLabels.tsx]: In GH is the same - take a look at tags cloud on our project repo

**Review by arkadius — APPROVED** [review]
One minor, not important comment. Feel free to ignore it

File: designer/client/src/components/toolbars/scenarioDetails/ScenarioLabels.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Dzuming) [reply designer/client/src/components/toolbars/scenarioDetails/ScenarioLabels.tsx]: Done ✔️

**Pull request merged**